lib/Runtime/Library/JavascriptFunction.cpp

```diff
@@ -691,10 +697,10 @@ namespace Js
         __finally
         {
             // 0xE06D7363 is C++ exception code
-            if (exceptionCode != 0 && !IsDebuggerPresent() && exceptionCode != 0xE06D7363 && exceptionAction != EXCEPTION_CONTINUE_EXECUTION)
+            if (AbnormalTermination() && exceptionCode != 0 && !IsDebuggerPresent() && exceptionCode != 0xE06D7363)
```

Comment:
>AbnormalTermination [](start = 16, length = 19)

Good one but does this have performance impact?
[https://msdn.microsoft.com/en-us/library/windows/desktop/ms679265(v=vs.85).aspx](https://msdn.microsoft.com/en-us/library/windows/desktop/ms679265(v=vs.85).aspx)